Packs/GCP/Integrations/GCP/GCP.yml

```diff
@@ -0,0 +1,1603 @@
+      type: number
+    - contextPath: GCP.Compute.Operations.httpErrorMessage
+      description: If the operation fails, this field contains the HTTP error message that was returned, such as NOT FOUND.
+      type: string
+    - contextPath: GCP.Compute.Operations.selfLink
+      description: Server-defined URL for the resource.
+      type: string
+    - contextPath: GCP.Compute.Operations.region
+      description: The URL of the region where the operation resides. Only available when performing regional operations. You must specify this field as part of the HTTP request URL. It is not settable as a field in the request body.
+      type: string
+    - contextPath: GCP.Compute.Operations.description
+      description: A textual description of the operation, which is set when the operation is created.
+      type: string
+    - contextPath: GCP.Compute.Operations.kind
+      description: Type of the resource. Always compute#operation for Operation resources.
+      type: string
+
+
+  - name: gcp-compute-instance-stop
+    description: Stops a running instance, shutting it down cleanly, and allows you to restart the instance at a later time. Stopped instances do not incur VM usage charges while they are stopped. However, resources that the VM is using, such as persistent disks and static IP addresses, will continue to be charged until they are deleted.
+    arguments:
+    - name: project_id
+      description: GCP project ID.
+      required: true
+    - name: zone
+      description: The name of the zone for this request.
+      required: true
+    - name: resource_name
+      description: Name of the instance resource to stop.
+      required: true
+    outputs:
+    - contextPath: GCP.Compute.Operations.id
+      description: The unique identifier for the resource. This identifier is defined by the server.
+      type: string
+    - contextPath: GCP.Compute.Operations.name
+      description: Name of the resource.
+      type: string
+    - contextPath: GCP.Compute.Operations.zone
+      description: The URL of the zone where the operation resides. Only available when performing per-zone operations. You must specify this field as part of the HTTP request URL. It is not settable as a field in the request body.
+      type: string
+    - contextPath: GCP.Compute.Operations.clientOperationId
+      description: The value of requestId if you provided it in the request. Not present otherwise.
+      type: string
+    - contextPath: GCP.Compute.Operations.operationType
+      description: The type of operation, such as insert, update, or delete, and so on.
+      type: string
+    - contextPath: GCP.Compute.Operations.targetLink
+      description: The URL of the resource that the operation modifies. For operations related to creating a snapshot, this points to the persistent disk that the snapshot was created from.
```

Comment:
The URL of the resource that the operation modifies. For operations related to creating a snapshot, this points to the persistent disk from which the snapshot was created.